src/views/ContinuationOut.vue

```diff
@@ -88,6 +70,65 @@
               </div>
             </section>
 
+            <!-- Effective Date of Continuation -->
+            <section>
+              <header>
+                <h2>Effective Date of Continuation</h2>
+              </header>
+              <div :class="{ 'invalid-section': !effectiveDateValid && showErrors }" id="effective-date-section">
+                <EffectiveDate
+                  :class="{ 'invalid-component': !effectiveDateValid && showErrors }"
+                  class="pt-6 px-4"
+                  ref="effectiveDateRef"
+                  :initialEffectiveDate="initialEffectiveDate"
+                  :validateForm="showErrors"
+                  @update:effectiveDate="effectiveDate=$event"
+                  @valid="effectiveDateValid=$event"
+                />
+              </div>
+            </section>
+
+            <!-- Jurisdiction Information -->
+            <section>
+              <header>
+                <h2>Jurisdiction Information</h2>
+              </header>
+              <div
+                :class="{ 'invalid-section': !foreignJurisdictionValid && showErrors }"
+                id="jurisdiction-information-section"
+              >
+                <ForeignJurisdiction
```

Comment:
The uxpin says to show the Province/State dropdown initially (ie, when country is not selected).

Also, the uxpin labels the region menu, "Jurisdisction Province" or "Jurisdiction State". Check with Janis as what you've done might be OK.

Reply:
Regarding this, you're right Sev. I'll have to wait until next Monday since both Yui and Janis are away today.